kajiki/tests/test_xml.py

```diff
@@ -968,5 +968,46 @@ def test_code_error(self):
                 assert False
 
 
+class TestBracketsInExpression(TestCase):
+    def test_simple(self):
+        perform('<x>${\'ok\'}</x>', '<x>ok</x>')
+
+    def test_some_brackets(self):
+        perform('<x>${\'{ok}\'}</x>', '<x>{ok}</x>')
+
+    def test_brackets_asymmetric(self):
+        perform('<x>${\'{o{k}k  { \'}</x>', '<x>{o{k}k  { </x>')
+
+    def test_complex(self):
+        perform(u"<xml><div>${'ciao {  } {' + \"a {} b {{{{} w}}rar\"}${'sd{}'} ${1+1}</div></xml>",
+                u"<xml><div>ciao {  } {a {} b {{{{} w}}rarsd{} 2</div></xml>")
+
+    def test_with_padding_space(self):
```

Comment:
Please add a case with a multiline expression to see that it reports an `IndentationError` instead of crashing with some unknown random error. So that user at least knows that it's a problem with indentation.